providers.tf

```diff
@@ -21,4 +21,10 @@ provider "consul" {
   address    = hcp_consul_cluster.main.consul_public_endpoint_url
   datacenter = hcp_consul_cluster.main.datacenter
   token      = hcp_consul_cluster_root_token.token.secret_id
-}
\ No newline at end of file
+}
+
+# Replace client_id and client_secret from values from HCP IAM settings 
+provider "hcp" {
+  client_id     = "service-principal-key-client-id"
+  client_secret = "service-principal-key-client-secret"
+}
```

Comment:
ill update the tutorial to export the HCP environment variables! this is the preferred way to set provider credentials

Reply:
Ok will remove thanks!